src/logger/MultiLogger.ts

```diff
@@ -32,9 +32,20 @@ export default class MultiLogger implements Logger {
     }
   }
 
-  debug(debugFunction: () => string): void {
+  debug(debugFunction: string | (() => string)): void {
+    let v: string;
```

Comment:
nit: Could we change this v to better variable name?

Reply:
Changed to `message`. Good call.